keyboards/handwired/tractyl_manuform/5x6_right/arduinomicro/config.h

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define PRODUCT Tractyl Manuform(5x6) Aruino Micro (r)/ Pro Micro (l)
```

Comment:
Just a note here, this will need additional changes in develop to wrap it in `""`, since the USB manufacturer and product strings are now string literals.

Reply:
I wrapped the whole define in `""`,  hope this is right?

Reply:
It will be correct in develop, but on master the device name will include the ""s, at least until the next breaking changes merge at the end of November.